styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,15 @@
-.footer{
-  
-}
\ No newline at end of file
+.rug{
+  .text37{
+    font-size: 37px;
+    line-height: 24px;
+  }
+  &, *{
```

Comment:
@bighappyface I made the changes to the PR we talked about except this one. I'm not sure what is a better way to make all the text within the rug white aside from adding the class white to every element. This way seems to be less repetitive and more encompassing. Do you have any alternatives?